alcs-frontend/src/app/features/notice-of-intent/decision/decision-v2/decision-input/decision-file-upload-dialog/decision-document-upload-dialog.component.ts

```diff
@@ -0,0 +1,103 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { DecisionDocumentDto } from '../../../../../../services/application/decision/application-decision-v2/application-decision-v2.dto';
```

Comment:
I know that model identical but I think they easer should be moved to common or there should be a separate model for NOI

Reply:
There is a separate one for NOIs, I've renamed the Application one to ApplicationDecisionDocumentDto and fixed the import here.